tests/unit/src/commands/clash.spec.js

```diff
@@ -29,7 +29,7 @@ describe('mtg clash command', function () {
     describe('clash', function () {
         describe('when', function () {
             it('should', function () {
-                expect(true).to.be.truthy();
```

Comment:
AFIACT, the ".truthy()" function does not exist within (Sinon-)Chai.

Standard Chai syntax is using the ".ok" prop to establish "truthiness"
http://chaijs.com/api/bdd/#ok